docs/5.x/requirements.md

```diff
@@ -88,27 +88,43 @@ These ports are required during initial installation and can be closed after the
-| 7496, 7373              | TCP                                     | Serf (Health check agents) peer to peer   |
-| 7575                    | TCP                                     | Cluster status gRPC API                   |
-| 2379, 2380, 4001, 7001  | HTTPS                                   | Etcd server communications                |
-| 6443                    | HTTPS                                   | Kubernetes API Server                     |
-| 30000 - 32767           | HTTPS (depend on the services deployed) | Kubernetes internal services range        |
-| 10248 - 10250, 10255    | HTTPS                                   | Kubernetes components                     |
-| 5000                    | HTTPS                                   | Docker registry                           |
-| 3022-3025               | SSH                                     | Teleport internal SSH control panel       |
-| 3080                    | HTTPS                                   | Teleport Web  UI                          |
-| 3008-3012               | HTTPS                                   | Internal Gravity services                 |
-| 32009                   | HTTPS                                   | Gravity Cluster/OpsCenter Admin panel UI  |
-| 3012                    | HTTPS                                   | Gravity RPC  agent                        |
+| Port                    | Protocol - Layer 4 | Protocol - Layer 5              | Source      | Destination | Description                              |
+|-------------------------|--------------------|---------------------------------|-------------|-------------|------------------------------------------|
+| 53                      | TCP/UDP            | DNS                             | localhost   | localhost   | Internal cluster DNS                     |
+| 8472                    | UDP                | VXLAN                           | all         | all         | Overlay network                          |
+| 7496                    | TCP/UDP            | HTTPs                           | all         | all         | Serf (Health check agents) peer to peer  |
+| 7373                    | TCP                | RPC                             | localhost   | localhost   | Serf RPC - peer to peer                  |
+| 7575                    | TCP                | gRPC                            | all         | all         | Cluster status gRPC API                  |
+| 2379, 2380, 4001, 7001  | TCP                | HTTPS                           | all         | controllers | Etcd server communications               |
+| 6443                    | TCP                | HTTPS                           | all         | controllers | Kubernetes API Server                    |
+| 30000 - 32767           | N.A                | depend on the services deployed | all         | all         | Kubernetes internal services range       |
```

Comment:
Depend**s**

Or I would rather rephrase to "Application specific". Also, I think it's "N/A", not "N.A" usually.

Reply:
It should be also noteworthy, that we recently removed enforcement of this range as kubernetes is using in on a best-effort basis (i.e. selects unoccupied ports), so it might be taken out from the list of hard requirements.

Reply:
The ports are still important to be listed in some way for AWS / Load balancer integrations, since the removal of the satellite check is a bit different. Even if kubernetes can bind to the port, doesn't mean the firewall allows the port to be used.